content/cli/users.md

```diff
@@ -149,21 +151,19 @@ User removed
 #### Usage 
 `cfy users list`
 
-List all users defined in this Cloudify Manager.<br>
-By default, when you generate the list of users, only the number of linked resources are displayed. You can retrieve full details with the use of a `--get-data` flag.
+In a non-LDAP setup this command will list all users defined in this Cloudify Manager. In LDAP mode, it will list all users who have performed a log in into cloudify and were successfully authenticated against the LDAP-based system.<br>
```

Comment:
Avoid future tense and "perform"
In non-LDAP mode, this command lists all of the users defined in this Cloudify Manager. In LDAP mode, this command lists all of the users who logged in to Cloudify and successfully authenticated with the LDAP system.